instagram/src/App.js

```diff
@@ -0,0 +1,7 @@
+import "./css/main.css";
+import authenticate from "./authenticate/authenticate";
+import Login from "./components/Login/Login";
```

Comment:
one thing to note: you actually spelled login.js in your file system with lowercase and not uppercase. more case-sensitive OS like Linux will fail to resolve this path unless fixed